README.md

```diff
@@ -0,0 +1,84 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Module 2 - To Do List
+
+HTML + JS + CSS project to implement an storage for a ToDo List.
+
+The project is designed to implement SPA concept, with just 1 HTML page and sections visible or hidden depending on the functionality selected.
+
+The functionalities are:
+
+- The Landing Page presenting the tasks stored in localStore.
+- Add a new task.
+- Modify a task.
+- Know the total active task in the red badge on top right of the app.
+- Remove one task.
+- Mark some tasks as active or completed.
+- Remove all the completed tasks.
+
+## Built With
+
+- HTML
+- JavaScript
+- CSS
+
+- Webpack
+- Fontawesome package installed by npm command.
+
+## Live Demo
+
+Not available yet.
+
+## Getting Started
+
+### Install
+
+- Create a folder into your storage device.
+- Open a command line Terminal and go to the new folder.
+- Execute: 'git clone git@github.com:lmunozm1702/to-do-list.git'
+
+- install and configure webpack - use [this](./webpack.sh) reference
+
+- execute npm start to see the homepage
+
+### Usage
+
+Once on the landing page, you will see the saved data displayed into the list and a badge with the total uncompleted tasks.
+
+You can create new elements clicking into the "Add to your list..." field.
+
+Also you can modify elements clicking directly into them. And remove it using the trash can icon visible when you click the task to edit.
+
+Mark and unmark a task with the checkbox in each one.
+
+remove all of the completed task, using the Clear all link at bottom of the app.
+
+### Deployment
+
+Not available yet.
+
```

Comment:
I think it would be better to add a live version demo of your project in the readme. You can use GitHub pages and it would look more professional since your visitors would be able to see your app without having to download your code.